docs/advanced-features/preview-mode.md

```diff
@@ -223,6 +223,10 @@ export default function myApiRoute(req, res) {
 }
 ```
 
+### Works with the router
```

Comment:
Maybe this could be a note that links to the router fields docs instead, what do you think?

Reply:
Sounds good to me as that page links back to this one too, thanks for the suggestion. Feel free to push a change as I'm away from the computer now.